src/core/ComplexFilter/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { AutocompleteCloseReason } from "@material-ui/lab";
+import React, { useEffect, useState } from "react";
+import InputDropdown, {
+  InputDropdownProps as InputDropdownPropsType,
+} from "../InputDropdown";
+import MenuSelect, { DefaultMenuSelectOption } from "../MenuSelect";
+import Chips from "./components/Chips";
+import { StyledPaper, StyledPopper, Wrapper } from "./style";
+
+export {
```

Comment:
Expose `ComplexFilterPopper` and `ComplexFilterStyledPaper`, so the callsite can further style the components and pass them back to ComplexFilter like so:

```tsx
import ComplexFilter, { ComplexFilterPopper } from 'czifui'

const CustomizedPopper = styled(ComplexFilterPopper)`
  ...
`

<ComplexFilter PopperComponent={CustomizedPopper} />
```

MUI uses this pattern in their components too!